.circleci/config.yml

```diff
@@ -286,6 +286,15 @@ jobs:
           export RELEASE_APPROVAL_TOKEN=$REPO_GITHUB_TOKEN
           bazel run @graknlabs_build_tools//ci:release-approval
 
+  release-validate:
+    machine: true
+    steps:
+      - install-bazel-linux-rbe
+      - checkout
+      - run: |
+          bazel run @graknlabs_build_tools//ci:release-validate-deps graknlabs_common \
```

Comment:
Do we not always need to provide arguments after `--` with `bazel run`?

Reply:
Can we have all the arguments in the next line? It'll just read better.

Reply:
Both addressed in https://github.com/graknlabs/grakn/pull/5439/commits/38f962b87dfceccc9097eb855c7842f126ccc980